packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -936,10 +936,16 @@ describe('DirectStripeRoutes', () => {
       directStripeRoutesInstance.stripeHelper.customer
         .onCall(1)
         .resolves(expected);
-
+      directStripeRoutesInstance.stripeHelper.customer.resolves({ data: [] });
```

Comment:
Why is this necessary?  I don't see an additional call to `stripeHelper.customer` above.